functions.php

```diff
@@ -302,13 +302,29 @@ function render_related_posts_block(array $attributes): string
     // Encode the query attributes to JSON for the block template
     $query_json = wp_json_encode($attributes['query_attributes'], JSON_UNESCAPED_SLASHES);
 
+    // Dynamically render link to News & Stories page
+    $news_stories_url = '';
+    $news_stories_page = get_posts([
```

Comment:
This is not the correct way to get the News & Stories page, since the title can be changed. You can get the id the same way we do in other places in the code such as [here](https://github.com/greenpeace/planet4-master-theme/blob/main/src/ListingPage.php#L101), like this: 
```
(int) get_option('page_for_posts')
```